cdk/test/cdk.test.ts

```diff
@@ -1,13 +1,22 @@
-import { expect as expectCDK, matchTemplate, MatchStyle } from '@aws-cdk/assert';
-import * as cdk from '@aws-cdk/core';
-import * as Cdk from '../lib/cdk-stack';
+import {
+  expect as expectCDK,
+  matchTemplate,
+  MatchStyle,
+} from "@aws-cdk/assert";
+import * as cdk from "@aws-cdk/core";
+import * as Cdk from "../lib/cdk-stack";
 
-test('Empty Stack', () => {
-    const app = new cdk.App();
-    // WHEN
-    const stack = new Cdk.CdkStack(app, 'MyTestStack');
-    // THEN
-    expectCDK(stack).to(matchTemplate({
-      "Resources": {}
-    }, MatchStyle.EXACT))
+xtest("Empty Stack", () => {
```

Comment:
Is `xtest` a typo?

Reply:
Nope, I mentioned I wanted to skip this test for now as a future PR will be added to add the real snapshot test.

Reply:
Argh! Sorry, should have read the description closer!